tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/stepdefinitions/throttling/ThrottlingStepDefs.java

```diff
@@ -0,0 +1,49 @@
+package io.hawt.tests.features.stepdefinitions.throttling;
+
+import com.codeborne.selenide.SelenideElement;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.hawt.tests.features.pageobjects.pages.LoginPage;
+import io.hawt.tests.features.setup.LoginLogout;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selenide.$;
+
+public class ThrottlingStepDefs {
+    private final LoginPage loginPage = new LoginPage();
+    private final static SelenideElement warning = $("p.pf-c-form__helper-text.pf-m-error");
+
+
+    @Given("User is on Login page")
+    public void userIsOnLoginPage() {
+        LoginLogout.logout();
+        loginPage.loginPageIsOpened();
+    }
+
+    @When("the user attempts to log in with incorrect credentials {int} times")
+    public void theUserAttemptsToLogInWithIncorrectCredentialsTimes(int attempts) {
+        for (int i = 0; i < attempts; i++) {
+            loginPage.throttling("username", "invalid");
+        }
+
+    }
+
+    @Then("the user should see a message indicating account lockout")
+    public void theUserShouldSeeAMessageIndicatingAccountLockout() {
+        warning.shouldHave(text("Login attempt blocked. Retry after 1 second"));
+    }
+
+    @When("the user attempts to log in with incorrect credential {int} times")
+    public void theUserAttemptsToLogInWithIncorrectCredentialTimes(int attempts) {
+        for (int i = 0; i < attempts; i++) {
+            loginPage.throttling("username", "invalid");
+        }
+    }
+
+    @Then("the user should see a message indicating account lockout second time")
+    public void theUserShouldSeeAMessageIndicatingAccountLockoutSecondTime() {
+        warning.shouldHave(text("Login attempt blocked. Retry after 3 seconds"));
+    }
+}
```

Comment:
Please make this a universal step "the user should see a message indicating account lockout for {int} seconds" and format the text condition to contain the int parameter.